src/hooks/useAuthorityLinking/useAuthorityLinking.js

```diff
@@ -242,6 +251,53 @@ const useAuthorityLinking = () => {
     };
   }, []);
 
+  const actualizeNewLinkedFields = useCallback(async (formValues) => {
+    const isNewLinkedField = (field) => field.linkDetails?.status === AUTOLINKING_STATUSES.NEW;
+    const hasNewLinkedField = formValues.fields.some(isNewLinkedField);
+
+    if (!hasNewLinkedField) {
+      return formValues;
+    }
+
+    const newLinkedFields = formValues.fields.filter(isNewLinkedField);
+    const payload = hydrateForLinkSuggestions(formValues, newLinkedFields);
+    const { fields: suggestedFields } = await fetchLinkSuggestions({
+      body: payload,
+      isSearchByAuthorityId: true,
+    });
+
+    let suggestedFieldIndex = 0;
+
+    const actualizedFields = formValues.fields.map(field => {
+      if (isNewLinkedField(field)) {
+        const suggestedField = suggestedFields[suggestedFieldIndex];

```

Comment:
you can use `suggestedFields.shift()` to get the first value - I think it will look cleaner without indexes and increments